doc/OnlineDocs/getting_started/index.rst

```diff
@@ -1,9 +1,58 @@
 Getting Started
 ===============
 
-Installation, pyomo command, a simple example, etc
+Installation
+------------
 
-.. doctest::
+Pyomo currently supports the following versions of Python:
+
+* CPython: 2.7, 3.5, 3.6
+
+
+Using CONDA
+~~~~~~~~~~~
+
+We recommend installation with *conda*, which is included with the Anaconda
+distribution of Python.  If you have a different Python distribution, then
+you can install *miniconda* using *pip*:
+
+.. bash::
+    pip install miniconda
+
+You can install Pyomo in your system Python installation
+by executing the following in a shell:
+
+.. bash::
+    conda install -c conda-forge pyomo
+
+Pyomo also has conditional dependencies on a variety of third-party Python packages.  These can also be installed with conda:
+
+.. bash::
+    conda install -c conda-forge pyomo.extras
+
+Optimization solvers are not installed with Pyomo, but some open source optimization solvers can be installed with conda as well:
+
+.. bash::
+    conda install -c conda-forge pyomo.solvers
+
+
+Using PIP
+~~~~~~~~~
+
+The standard utility for installing Python packages is *pip*.  You
+can install Pyomo in your system Python installation by executing
+the following in a shell:
+
+.. bash::
+    pip install pyomo
+
+However, *pip* does not support the robust installation of conditional
+dependencies (i.e. the third-party Python packages or the solvers
+that Pyomo may use).
```

Comment:
I disagree with this statement.  pip is the generally recommended installation method for Python packages.

Reply:
This is a true statement, regardless whether pip is generally recommended.  But I'll delete it.